### PR TITLE
tests: update failing trace webtest to use Tooltip module

### PR DESCRIPTION
**Pull request by connorjclark**
Changed here: https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2555060

I'd expect that change to fail this test https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/test/webtests/http/tests/devtools/lighthouse/lighthouse-view-trace-run.js;l=36;drc=b736005f57eccddf8dd024872552f1bfb0264682 but it didn't. @TimvdLippe I assumed `test/webtests` run in CQ, is that true?

### CODE REVIEW

**Comment by TimvdLippe** [comment]
The reason the test wasn't failing is that the getter was previously implicitly calling `UI.Tooltip.getContent`. The setter in turn was sugar for `UI.Tooltip.install`. Therefore, I migrated the setters first, without touching the getters. I fixed these tests when I removed the getter and setter in https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2562343

**Review by patrickhulce — APPROVED** [review]

**Comment by patrickhulce** [comment]
@connorjclark FYI this is failing the devtools check with `UI.Tooltip.getContent is not a function`. Something need to be updated to match ToT?

**Comment by connorjclark** [comment]
weird, it works for me locally even after I clear `.tmp`. Let me try to invalidate the cache here.

File: .github/workflows/devtools.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
           ${{ env.DEVTOOLS_PATH }}
           ${{ env.BLINK_TOOLS_PATH }}
           ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
-        key: ${{ runner.os }}-${{ hashFiles('lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
+        key: ${{ runner.os }}-${{ hashFiles('third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/*.js', 'lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
```
Comment (connorjclark) [inline .github/workflows/devtools.yml:36]: I assumed the test files were already apart of this hash. This should fix CI.

**Pull request merged**